cpg_workflows/jobs/gcnv.py

```diff
@@ -468,7 +456,7 @@ def run_joint_segmentation(
     tmp_prefix: str,
     output_path: Path,
     job_attrs: dict[str, str] | None = None,
-) -> list[Job]:
+) -> list[BashJob]:
```

Comment:
OOI, What does BashJob give us here over Job?

Reply:
Ah this was just some rubbish with the type inspections - we were setting an expected `Job` return, when we were generating a BashJob 🤷 

This doesn't fail in CI, but it fails for me locally - related to the whole 'we aren't doing proper type inspections' issue addressed [here](https://github.com/populationgenomics/production-pipelines/pull/728)